pkg/proxy/proxier.go

```diff
@@ -319,38 +319,43 @@ type Proxier struct {
 	hostIP        net.IP
 }
 
+var (
+	ErrProxyOnLocalhost = fmt.Errorf("cannot proxy on localhost")
```

Comment:
It was our only typed error (error we logged, vs propagating up from iptables).

----- Original Message -----
> > @@ -319,38 +319,43 @@ type Proxier struct {
> >  	hostIP        net.IP
> >  }
> >  
> > +var (
> > +	ErrProxyOnLocalhost = fmt.Errorf("cannot proxy on localhost")
> 
> Why is this one error promoted to a variable but no others?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/8269/files#r30357599